src/item.test.ts

```diff
@@ -565,4 +633,26 @@ describe('video elements', () => {
         const element = getFirstBody(item);
         expect(element.kind).toBe(ElementKind.Interactive);
     });
+});
+
+describe('format', () => {
+    test('get format returns correct fields', () => {
+        const item = f(immersive);
+        const format = getFormat(item);
+        expect('design' in format).toBe(true);
+        expect('display' in format).toBe(true);
+        expect('pillar' in format).toBe(true);
```

Comment:
I think this *should* be caught by the compiler, as `getFormat` returns `Format`.